src/App.js

```diff
@@ -15,17 +15,21 @@ export default function App() {
   return (
     <Router>
       <div>
-        <nav>
-          <ul>
-            <li>
-              <Link to="/">Books</Link>
-            </li>
-            <li>
-              <Link to="categories">Categories</Link>
-            </li>
-          </ul>
-        </nav>
-
+        <div className="header">
+          <div>
+            <h1 className="headerTitle">Bookstore Ben</h1>
+          </div>
+          <nav className="NavContainer">
+            <ul className="navBar">
+              <li>
+                <Link to="/" className="navItem">Books</Link>
+              </li>
+              <li>
+                <Link to="categories" className="navItem">Categories</Link>
+              </li>
+            </ul>
+          </nav>
+        </div>
```

Comment:
- [ ] Your app is expected to match the design of the Zeplin template. Kindly add the user icon to the right on the Navbar.

Your Implementation | Zeplin Design
:---:|:---:
![benjp-icon-marked](https://user-images.githubusercontent.com/36542986/150310623-99b38187-79ba-49ab-884f-167c93e3c2e0.jpg) | ![benjp-icon](https://user-images.githubusercontent.com/36542986/150310615-8d35a5a7-2313-47bb-a56e-42412749c5b2.jpg)